internal/runners/initialize/init.go

```diff
@@ -148,9 +148,24 @@ func (r *Initialize) Run(params *RunParams) (rerr error) {
 		}
 	}
 
+	// Match the case of the organization.
+	// Otherwise the incorrect case will be written to the project file.
+	owner := params.Namespace.Owner
+	orgs, err := model.FetchOrganizations()
+	if err != nil {
+		return errs.Wrap(err, "Unable to get the user's writable orgs")
+	}
+	for _, org := range orgs {
+		if strings.EqualFold(org.DisplayName, owner) {
```

Comment:
We may as well error out if we can't find a matching org. Otherwise, we'll error out with the API call further down anyway, but the user will have an activestate.yaml which can't be used.